creator-cards/aneesh-gopalakrishnan.json

```diff
@@ -1,26 +1,30 @@
 {
   "name": "Aneesh Gopalakrishnan",
-  "sectionTitle": null,
-  "countryCode": "us",
-  "slogan": "Nobody Special!",
+  "countryCode": "USA",
+  "slogan": "",
   "type": "CREATOR",
   "bio": "Engineer",
-  "tags": ".NET, react",
+  "tags": ".NET, C#, ASP.NET Core, react, Azure",
   "socials": {
     "linkedin": "https://www.linkedin.com/in/aneeshlalga",
     "twitter": "https://twitter.com/aneeshlalga",
     "github": "https://github.com/codehippie1"
   },
-  "channels": [
+  "feeds": [
+    {
+      "type": "LinkTagRssType",
+      "url": "https://codehippie1.github.io/aneeshlal.github.io/blogs.xml"
+    },
     {
-      "type": "Blog",
-      "url": "https://www.red-gate.com/simple-talk/development/angular/inline-pdf-viewer-in-an-angular-app-now-you-can/"
+      "type": "LinkTagRssType",
+      "url": "https://codehippie1.github.io/aneeshlal.github.io/talks.xml"
     }
   ],
-  "feeds": [
+  "projects": [
     {
-      "type": "Blog",
-      "url": "https://rss.app/feeds/79Dl0OyyexNBxDuA.xml"
+      "name": "ng2-pdfjs-viewer",
+      "description": "Angular PDF Viewer",
+      "url": "https://github.com/intbot/ng2-pdfjs-viewer"
```

Comment:
Can you tell me why this is related to .NET? Does not look like a .NET Project to me.